pkg/scheduler/flavorassigner/tas_flavorassigner.go

```diff
@@ -58,6 +60,19 @@ func (a *Assignment) WorkloadsTopologyRequests(wl *workload.Info, cq *cache.Clus
 	return tasRequests
 }
 
+func (psa *PodSetAssignment) HasFailedNodeAssignment(wl *workload.Info) bool {
```

Comment:
```suggestion
func (psa *PodSetAssignment) HasFailedNode(wl *workload.Info) bool {
```
nit, maybe the assignment is already obvious from the context?

Reply:
Done